testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -33,8 +33,7 @@
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.List;
-
-import javax.ws.rs.BadRequestException;
+import java.util.stream.Collectors;
```

Comment:
I wonder if there is a good time to possibly "decompose" ClientPoliciesTest into multiple test classes? For example one class, which will focus mostly on CRUD of clients and another with focus on stuff like signatures, token introspection etc? I have some impression that current class is quite big and already has 29 test methods.

Just an idea for the future. I suggest to *not* do it in this PR, but rather have separate PR for this...

Reply:
@mposolda Yes, I agree with you.  How about creating the follow-up JIRA ticket for refactoring `ClientPoliciesTest` after the following PR being merged?

- Client Policies
https://github.com/keycloak/keycloak/pull/7780

Reply:
@tnorimat +1 for creating follow-up JIRA. Thanks!

Reply:
@mposolda I've created [the JIRA ticket](https://issues.redhat.com/browse/KEYCLOAK-17353).